tkg/client/cluster.go

```diff
@@ -862,3 +874,55 @@ func (c *TkgClient) ValidateManagementClusterVersionWithCLI(regionalClusterClien
 
 	return nil
 }
+
+// validateControlPlaneTaint validates if controlPlaneTaint CC variable is not set for the single node workload cluster, otherwise returns error
+func validateControlPlaneTaint(stream []byte, options *CreateClusterOptions, tkgClient *TkgClient, regionalClient clusterclient.Client) error {
```

Comment:
suggestion: It seems like the function scope is more than validating the taint. Do you think it's better to rename it for e.g. `ValidateSNCCreation`?

Reply:
good point, but I am still thinking of a better name 😄

Reply:
Renamed to `validateConfigForSingleNodeCluster`